bigbone-rx/src/main/kotlin/social/bigbone/rx/RxAnnouncementMethods.kt

```diff
@@ -20,22 +20,15 @@ class RxAnnouncementMethods(client: MastodonClient) {
      * @see <a href="https://docs.joinmastodon.org/methods/announcements/#get">Mastodon API documentation: methods/announcements/#get</a>
      */
     @JvmOverloads
-    fun getAllAnnouncements(
-        withDismissed: Boolean = false
-    ): Single<List<Announcement>> = Single.fromCallable {
-        announcementMethods.getAllAnnouncements(withDismissed).execute()
-    }
+    fun getAllAnnouncements(withDismissed: Boolean = false): Single<List<Announcement>> =
+        Single.fromCallable { announcementMethods.getAllAnnouncements(withDismissed).execute() }
```

Comment:
I really don’t prefer the new version 😕

Reply:
Hey @PattaFeuFeu I also like the previous version better. @bocops any objections?

Reply:
Hard to tell actually - I generally don't like having a single parameter on its own line if not absolutely necessary, but in this specific case, both declarations are somewhat crowded due to everything else that is going on at the same time.

Are any less complicated declarations affected by this rule change that we could have a look at? If not, no strong objection to reverting this, though.